samples/FunctionApp/Program.cs

```diff
@@ -21,11 +18,10 @@ static async Task Main(string[] args)
             //<docsnippet_startup>
             var host = new HostBuilder()
                 //<docsnippet_configure_defaults>
-                .ConfigureFunctionsWorkerDefaults(builder =>
+                .ConfigureFunctionsWorkerDefaults((WorkerOptions options) =>
                 {
-                    builder
-                        .AddApplicationInsights()
-                        .AddApplicationInsightsLogger();
+                    // example (will remove before merge)
+                    options.Capabilities.Remove("HandlesWorkerTerminateMessage");
```

Comment:
We can add a unit test for this case if not done already.

Reply:
yes -- need to add some tests